README.md

```diff
@@ -284,7 +299,10 @@ spec:
         apiVersion: extensions/v1beta1
         kind: Ingress
         name: myapp
-      targetValue: 10 # this will be treated as targetAverageValue
+      averageValue: 10 # Only works with Kubernetes >=1.14
```

Comment:
It's called `targetAverageValue`

Reply:
Not here https://github.com/kubernetes/kubernetes/blob/a87e9a978f65a8303aa9467537aa59c18122cbf9/staging/src/k8s.io/api/autoscaling/v2beta1/types.go#L135 :)

It would make too much sense if the names matched across metric types 😂

Reply:
I don't understand why they have versioned betas if they keep changing all the versions. Oh well... 😄